backend/Application/Api/GraphQL/ChainParameters.cs

```diff
@@ -89,6 +89,37 @@ public override int GetHashCode()
         }
     }
 
+    /// <summary>
+    /// Present from protocol version 4 and above and hence from chain parameters 1 and above.
+    /// </summary>
+    internal static bool TryGetPassiveCommissions(
```

Comment:
I think it's worth being consistent in which flavor of switch cases we use. Elsewhere I've seen switch expressions.
Also it seems we're missing a test for this method.

Reply:
Good point added test :-)